### DISCUSSION TITLE
StockAlertBot on Target

### DISCUSSION THREAD

**Discussion by xmpierce1** [Prince25/StockAlertBot#58] [Q&A] [answered]
I am checking inventory for PlayStation 5 at Target and getting thrown an error that the browsers on the server need to be upgraded. They are all up to the current code base along with Windows 10. The error page is here.. Any ideas how to get around this?

![image](https://user-images.githubusercontent.com/74942520/112078920-a1533d00-8b55-11eb-9b2d-e0fd89e13710.png)

**Comment by Prince25** [comment]
Seems like they're blocking the requests. Does this happen once in a while or every single try?

**Comment by xmpierce1** [comment]
Every time ..

Sent from my mobile device


On Mar 22, 2021, at 11:26 PM, Prabhjot Singh ***@***.***> wrote:

﻿

Seems like they're blocking the requests. Does this happen once in a while or every single try?

**Reply by Prince25** [reply]
I will look into it.

**Reply by Prince25** [reply]
It seems to work just fine for me. May I see your `config.json` and the html file? 
![image](https://user-images.githubusercontent.com/28243638/112224747-fa1fe580-8be8-11eb-85fe-8eddd43790d9.png)

**Comment by xmpierce1** [comment]
See attached and thanks for checking this out. It runs fine the first pass but the second pass it fails..

**Reply by Prince25** [reply]
It seems like your files are not attached through email. Maybe try it through GitHub.

**Comment by xmpierce1** [comment]
See attached..


[config.json.txt](https://github.com/Prince25/StockAlertBot/files/6194452/config.json.txt)

**Reply by Prince25** [reply]
I tried your `config.json` and I had no issues. 
![image](https://user-images.githubusercontent.com/28243638/112387434-feaed180-8cae-11eb-9f80-9841c255673c.png)

Could you tell me your node version by running `node -v`? Also try deleting the `node_modules` folder and re-running `npm i`.

**Comment by xmpierce1** [comment]
I will. I am out of town until Sunday though . It’s running and target will drop tomorrow morning so I will see what happens if anything . Thanks for the fast response !

Sent from my mobile device


On Mar 24, 2021, at 3:42 PM, Prabhjot Singh ***@***.***> wrote:

﻿

I tried your config.json and I had no issues.
[image]<https://user-images.githubusercontent.com/28243638/112387434-feaed180-8cae-11eb-9f80-9841c255673c.png>

Could you tell me your node version by running node -v? Also try deleting the node_modules folder and re-running npm i.

**Reply by Prince25** [reply]
Sure no problem. So is it running without giving the error page you mentioned?

**Comment by xmpierce1** [comment]
Still the same message but we can address it next week. I just do not have remote access to the box.

**Reply by Prince25** [reply]
Alright. Just let me know.

**Comment by xmpierce1** [comment, marked as answer]
So the error still happens but the alerts still work. It caught the PS5 drop today


[cid:3FF7E754-FC32-49C0-960B-75AB2B87A618]

Sent from my mobile device


On Mar 25, 2021, at 12:26 AM, Prabhjot Singh ***@***.***> wrote:

﻿

Alright. Just let me know.

**Reply by Prince25** [reply]
As long as the program is still running and this error doesn't happen on every single check, then it's all good. If you're receiving alerts, then the error is probably not happening on every single check. If possible, could you provide the html file? Thanks!

**Comment by xmpierce1** [comment]
I will this weekend when I get back. The program works great overall, I like to see how I can have certain product alerts go to separate DISCORD channels like RTX Video Cards, etc… I admit I have not gone that deep. I also have Discord going to Twitter via Zapier, which is expensive and like a different solution.

**Reply by Prince25** [reply]
Sounds great! For Discord, you can copy the program and run separate instances. Each instance will have different URLs and the webhooks you'd want them to go to. In regards to Twitter, I put in some thought into if I want to integrate it and I concluded that it does not fit the intention of this program. This program is meant for personal use, not broadcast to an audience. However, I think you can easily find free solutions for sending notifications from Discord to Twitter.

I'm happy it worked out for you. If possible, please consider sharing the project and donating so I can continue working on this. 😄

**Comment by xmpierce1** [comment]
Absolutely!

Sent from my mobile device


On Mar 25, 2021, at 3:50 PM, Prabhjot Singh ***@***.***> wrote:

﻿

Sounds great! For Discord, you can copy the program and run separate instances. Each instance will have different URLs and the webhooks you'd want them to go to. In regards to Twitter, I put in some thought into if I want to integrate it and I concluded that it does not fit the intention of this program. This program is meant for personal use, not broadcast to an audience. However, I think you can easily find free solutions for sending notifications from Discord to Twitter.

I'm happy it worked out for you. If possible, please consider sharing the project and donating so I can continue working on this. 😄
